cmd/vulcan-aws-alerts/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/sirupsen/logrus"
+
+	check "github.com/adevinta/vulcan-check-sdk"
+	"github.com/adevinta/vulcan-check-sdk/state"
+	"github.com/aws/aws-sdk-go/aws/arn"
+)
+
+var (
+	checkName = "vulcan-aws-alerts"
+	logger    = check.NewCheckLog(checkName)
+)
+
+type options struct {
```

Comment:
Now you can pass required vars via local.toml without doing this trick:

https://github.com/adevinta/vulcan-check-sdk/pull/5